pkgs/top-level/haxe-packages.nix

```diff
@@ -19,39 +19,38 @@ let
 
     # simulate "haxelib dev $libname ."
     simulateHaxelibDev = libname: ''
-        export HOME=$(mktemp -d)
-        mkdir -p $HOME/devrepo/${libname}
-        echo $(pwd) > $HOME/devrepo/${libname}/.dev
-        export HAXELIB_PATH="$HAXELIB_PATH:$HOME/devrepo"
-      '';
+      export HOME=$(mktemp -d)
+      mkdir -p $HOME/devrepo/${libname}
+      echo $(pwd) > $HOME/devrepo/${libname}/.dev
+      export HAXELIB_PATH="$HAXELIB_PATH:$HOME/devrepo"
+    '';
 
     installLibHaxe = { libname, version, files ? "*" }: ''
-        mkdir -p "$out/lib/haxe/${withCommas libname}/${withCommas version}"
-        echo -n "${version}" > $out/lib/haxe/${withCommas libname}/.current
-        cp -dpR ${files} "$out/lib/haxe/${withCommas libname}/${withCommas version}/"
-      '';
-
-    defaultMeta = libname: {
-        homepage = "http://lib.haxe.org/p/${libname}";
-        license = stdenv.lib.licenses.bsd2;
-        platforms = stdenv.lib.platforms.all;
-        description = throw "please write meta.description";
-      };
-
-    fetchHaxeLib = { libname, version, sha256, meta, postFixup ? "", propagatedBuildInputs ? [] }:
-      stdenv.mkDerivation {
-        inherit propagatedBuildInputs postFixup;
+      mkdir -p "$out/lib/haxe/${withCommas libname}/${withCommas version}"
+      echo -n "${version}" > $out/lib/haxe/${withCommas libname}/.current
+      cp -dpR ${files} "$out/lib/haxe/${withCommas libname}/${withCommas version}/"
+    '';
+
+    buildHaxeLib = {
```

Comment:
The difference is it does not build, it downloads and unzip precompiled library.
So I named in ```fetch*``` thinking about functions like ```fetchMavenArtifact```